recipes/dlib/all/conanfile.py

```diff
@@ -90,91 +81,85 @@ def requirements(self):
             self.requires("openblas/0.3.20")
 
     def validate(self):
-        if is_msvc(self) and self.options.shared:
```

Comment:
Let's check what is failing and if we can fix with Conan 2.x. I tested locally in Mac M1 and I can build, but I'm using armv8 -> armv8